pyaedt/modules/CircuitTemplates.py

```diff
@@ -0,0 +1,580 @@
+# Power Sinusoidal
+PowerSinusoidal = [
+    "NAME:Name",
```

Comment:
I would keep it because we use the template to send directly the command. In case of the API changes, it will be easier to detect the problem. This 'NAME' input is a dummy and is always the same.
@maxcapodi78